tools/dev/get_release_artifacts.py

```diff
@@ -0,0 +1,69 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import datetime
+import subprocess
+import shutil
+import sys
+
+from dateutil import parser, tz
+
+
+def main():
+    gh_bin = shutil.which("gh")
+    assert gh_bin, "Please install gh cli https://cli.github.com"
+    command = [
+        gh_bin,
+        "run",
+        "list",
+        "-w",
+        "release.yml",
+        "--json",
+        "status",
+        "--json",
+        "workflowName",
+        "--json",
+        "updatedAt",
+        "--json",
+        "databaseId",
+        "--json",
+        "conclusion",
+        "--json",
+        "headBranch",
+        "--json",
+        "headSha",
```

Comment:
TIL you can pass `--json` multiple times instead of comma-separating properties in a single value. Neat :+1:

Reply:
that would be shorter, thanks, 👍